tests/api_gov.spec.js

```diff
@@ -360,4 +361,36 @@ describe('TESTING /v2/gov/new zealand', () => {
         });
     });
   });
-});
+
+describe('TESTING /v2/gov/south africa', () => {
+    it('/v2/gov/south africa correct data', (done) => {
+      chai.request(app)
+        .get('/v2/gov/south africa')
+        .end((err, res) => {
+          testBasicProperties(err, res, 200, 'object');
+          res.body.should.have.property('updated');
+          res.body.should.have.property('national');
+		  res.body.should.have.property('provinces');
+		  
+		  res.body.national.timeline.length.should.be.at.least(107);
+		  res.body.national.timeline[90].date.should.equal('2020-06-02');
+		  res.body.national.timeline[90].cases.cumulative.should.equal(35812);
+		  res.body.national.timeline[90].cases.unallocated.should.equal(6);
+		  res.body.national.timeline[90].cases.new.should.equal(1455);
+		  res.body.national.timeline[90].tests.cumulative.should.equal(761534);
+		  res.body.national.timeline[90].recoveries.cumulative.should.equal(18313);
+		  res.body.national.timeline[90].deaths.cumulative.should.equal(755);
+		  res.body.national.timeline[90].deaths.new.should.equal(50);
+
+		  res.body.provinces.length.should.equal(9);
+		  res.body.provinces[0].name.should.equal('Eastern Cape');
+		  res.body.provinces[0].timeline.length.should.be.at.least(107);
+		  res.body.provinces[0].timeline[85].date.should.equal('2020-05-28');
+		  res.body.provinces[0].timeline[85].cases.should.equal(3306);
```

Comment:
this seems kinda odd, why is it index 90 and how can you know it will stay there? same for provinces (85).

Reply:
The data from 90 days into data tracking won't change, since it's in the past, I am assuming. Better to have dynamic tests still though, so we don't have failing tests from a source update.

Reply:
Yeah, the purpose here is to check that the correct data gets scraped for that day 28 May 2020. This is essentially just a spot check to catch scenarios where the webpage might change its structure and the column indexes go out of sync causing wrong values to be used within the properties.
I will make it more dynamic by finding the province object where name == 'Eastern Cape' rather than using hardcoded indexes. I'll do the same for the other instances in the test as well.